### PR TITLE
Service/adds hastebin

### PR DESCRIPTION
**Pull request by denis-ev**
#119 adds hastebin

### CODE REVIEW

File: roles/vivumlab_config/templates/config.yml [hunk 1]
```diff
@@ -982,7 +993,7 @@ nzbget:
 octoprint:
   enable: {{ octoprint.enable | default(False) }}
   https_only: {{ octoprint.https_only | default(False) }}
-  devices: {{ octoprint.devices | default([])}}
+  devices: !!str '{{ octoprint.devices | default([]) }}'
```
Comment (voryzen) [inline roles/vivumlab_config/templates/config.yml:996]: Was this supposed to be there?

File: tasks/AddPackageTasks.sh [hunk 2]
```diff
@@ -1,84 +1,95 @@
...
-#   git branch $1
-
-#   git checkout $1
-# }
-
-# function search_and_replace_in_file(){
-#   sed -i "s~$1~$2~g" $3
-# }
+
+ Task::add_package() {
```
Comment (voryzen) [inline tasks/AddPackageTasks.sh:6]: Shouldn't this be in its own PR?

**Review by voryzen — DISMISSED** [review]

File: tasks/AddPackageTasks.sh (hunk 2 above)
Comment (denis-ev) [reply tasks/AddPackageTasks.sh:6]: Good catch, yes it should be.

File: roles/vivumlab_config/templates/config.yml (hunk 1 above)
Comment (denis-ev) [reply roles/vivumlab_config/templates/config.yml:996]: Well this threw an error, so I had to include it.

Comment (voryzen) [reply roles/vivumlab_config/templates/config.yml]: I figured this should be in its own PR, so the error is documentated.

But we are also cleaning up the changelog, so I'm not sure what the go is

Comment (denis-ev) [reply roles/vivumlab_config/templates/config.yml]: This is outdated and not even accurate any more. haha. 
It's not in this PR!

**Review by voryzen — APPROVED** [review]

File: group_vars/all [hunk 3]
```diff
@@ -1,4 +1,3 @@
----
```
[the remark this answers is not in the document]
Comment (voryzen) [reply group_vars/all:1]: Waiting on your reply, @denis-ev

File: tasks/AddPackageTasks.sh [hunk 4]
```diff
@@ -55,57 +53,83 @@
   highlight "Adding Docs to mkdocs.yml"
   cat package_template/mkdocs.yml > package_template/tmpmkdocsfile.yml
 
-  # Edit the config tempfile
```
[the remark this answers is not in the document]
Comment (voryzen) [reply tasks/AddPackageTasks.sh:58]: Waiting on your reply, @denis-ev

**Review by voryzen — CHANGES_REQUESTED** [review]
Waiting on your resolutions, @denis-ev

File: group_vars/all (hunk 3 above)
[the remark this answers is not in the document]
Comment (denis-ev) [reply group_vars/all:1]: yes, those dashes are not supposed to be there.

**Review by voryzen — APPROVED** [review]

**Pull request merged**